packages/wrangler/src/index.tsx

```diff
@@ -1121,27 +1123,24 @@ export async function main(argv: string[]): Promise<void> {
               });
           },
           async (args) => {
-            if (args._.length !== 2) {
-              throw new Error(
-                `Did you forget to add quotes around "${
-                  args.namespace
-                } ${args._.slice(2).join(" ")}"?`
-              );
+            if (args._.length > 2) {
+              const extraArgs = args._.slice(2).join(" ");
+              throw `Unexpected additional positional arguments "${extraArgs}".`;
             }
+
+            validateNamespaceBinding(args.namespace);
```

Comment:
Do you want an abstraction for a single usage? Maybe this should stay inlined for a bit.

Reply:
Here's my take on this:

- Functions are not just for DRYness, they are also valuable as self-documenting code. We don't really care that this is doing a regular expression check on the namespace at this point in the code. We just know that the namespace must be valid.
- Putting this code in a function allows us to move what is very much a private implementation detail of kv-namespace bindings closer to the rest of the kv code, hiding it from the reader here lowers the effort to understand the code here.

Happy to debate this if you like.

Reply:
> Putting this code in a function allows us to move what is very much a private implementation detail of kv-namespace bindings closer to the rest of the kv code, hiding it from the reader here lowers the effort to understand the code here.

This could be a valid point for colocation.